bench.py

```diff
@@ -197,6 +201,12 @@ def parse_args(parser, size=None, loop_types=(),
         except ImportError:
             print('Failed to import daal4py.sklearn.patch_sklearn.'
                   'Use stock version scikit-learn', file=sys.stderr)
+            params.device = 'host'
+    else:
+        if params.device != 'host':
+            print('Device context not supported without intel optimized version',
```

Comment:
Device context is not supported for stock scikit-learn. Please use `--no-intel-optimized=False` with f'--device={params.device}' parameter. Fallback to `--device=None`.